basetest.pm

```diff
@@ -28,8 +28,10 @@ use testapi  ();
 use autotest ();
 use MIME::Base64 'decode_base64';
 use Mojo::File 'path';
+use Try::Tiny;
```

Comment:
You will need to update the cpanfile and likely the os-autoinst.spec file too

Reply:
We're already using this all over the place - also in os-autoinst. `Try::Tiny` is missing in the cpan and spec file so it appears it is already pulled by some dependency.